lib/cassandra/cql.lua

```diff
@@ -1377,7 +1380,16 @@ do
     elseif op_code == OP_CODES.ERROR then
       local code = body:read_int()
       local message = body:read_string()
-      return nil, '['..ERROR_TRANSLATIONS[code]..'] '..message, code
+      local error_translation = ERROR_TRANSLATIONS[code]
+
+      -- If the translation is not found, return a formatted string
+      -- with the error code for convenience.
+      if error_translation == nil then
+        error_translation = string.format(
+          'UNSUPPORTED ERROR (code=%d)', code or ERRORS.UNKNOWN)
```

Comment:
I don't think we should use the `%d` modifier here - using `fmt("0x%x", code)` will print the error code in the same format the binary protocol specification refers to it.